cmd/export-distro/res/configuration.toml

```diff
@@ -52,6 +52,7 @@ Protocol = 'tcp'
 Host = 'localhost'
 Port = 5563
 Type = 'zero'
+Topic = 'events'
```

Comment:
Needs to be added to `cmd/export-distro/res/docker/configuration.toml` also

Reply:
will do